plugins/storage/volume/solidfire/src/main/java/org/apache/cloudstack/storage/datastore/lifecycle/SolidFireSharedPrimaryDataStoreLifeCycle.java

```diff
@@ -245,7 +243,7 @@ public DataStore initialize(Map<String, Object> dsInfos) {
 
         details.put(SolidFireUtil.VOLUME_ID, String.valueOf(sfVolume.getId()));
 
-        parameters.setUuid(iqn);
+        parameters.setUuid(UUID.randomUUID().toString());
```

Comment:
so this is the only function change in this file amongst a lot of good cleanup. I known not enough of solidfire to judge its merits. Is there any chance you can split reformatting and functional changes to better my review capabilities?

Reply:
So, honestly, here's the thing (and this may not be a good answer, I understand):

We waste so much time with PRs in verifying that failed tests are, in fact, false calls that I just decided to coalesce this minor update with the other changes. I know it's ideal to have each and every individual unit of functionality, in theory, checked in separately, but, in practice, it's a huge time drain.

Reply:
I am not talking separate PRs, @mike-tutkowski . Only separate commits.